api/src/main/java/org/hawkular/inventory/api/filters/Related.java

```diff
@@ -81,6 +82,17 @@
     }
 
     /**
+     * Creates a filter for entities that are sources of at least one relationship with the specified id. The target
+     * entity is not specified and can be anything.
+     *
+     * @param edgeId the id of the edge
+     * @return a new "related" filter instance
+     */
+    public static Related<?> byEdgeWithId(String edgeId) {
```

Comment:
We have entities and relationships. Edge is an implementation detail in tinkerpop.
In another words, I'd call this `byRelationshipWithId(String id)`.